Packs/ExpanseV2/Integrations/ExpanseV2/ExpanseV2.py

```diff
@@ -1589,23 +1612,46 @@ def get_remote_data_command(client: Client, args: Dict[str, Any], sync_owners: b
                 demisto.debug(f'The user assigned to Expanse incident {parsed_args.remote_incident_id} [{new_value}]'
                               f'is not registered on XSOAR, cannot change owner')
 
-        # handle issue closure
-        elif update_type == 'ProgressStatus' and new_value in ISSUE_PROGRESS_STATUS_CLOSED:
-            close_reason = EXPANSE_RESOLVEDSTATUS_TO_XSOAR[
-                new_value] if new_value in EXPANSE_RESOLVEDSTATUS_TO_XSOAR else 'Other'
-            resolve_comment = latest_comment['value'] if 'value' in latest_comment else ''
-            demisto.debug(f'Closing Expanse issue {parsed_args.remote_incident_id}')
+        elif update_type == 'ProgressStatus':
+            # handle issue closure
+            if previous_value not in ISSUE_PROGRESS_STATUS_CLOSED and new_value in ISSUE_PROGRESS_STATUS_CLOSED:
+                close_reason = EXPANSE_RESOLVEDSTATUS_TO_XSOAR[
+                    new_value] if new_value in EXPANSE_RESOLVEDSTATUS_TO_XSOAR else 'Other'
+                resolve_comment = latest_comment['value'] if 'value' in latest_comment else ''
+                demisto.debug(f'Closing Expanse issue {parsed_args.remote_incident_id}')
+                new_entries.append({
+                    'Type': EntryType.NOTE,
+                    'Contents': {
+                        'dbotIncidentClose': True,
+                        'closeReason': close_reason,
+                        'closeNotes': resolve_comment
+                    },
+                    'ContentsFormat': EntryFormat.JSON
```

Comment:
you could also use `CommandResults` here but not sure it will be better
```suggestion
                        'closeNotes': resolve_comment,
                    },
                    'ContentsFormat': EntryFormat.JSON,
```